core/src/main/resources/context-rule-description/others_section_html_content.html

```diff
@@ -2,9 +2,9 @@ <h4>How can I fix it in another component or framework?</h4>
 <p>Although the main framework or component you use in your project is not listed, you may find helpful content in the instructions we provide.</p>
 <p>Caution: The libraries mentioned in these instructions may not be appropriate for your code.</p>
 <p>
-<ul>
-    <li>Do use libraries that are compatible with the frameworks you are using.</li>
-    <li>Don't blindly copy and paste the fix-ups into your code.</li>
+<ul class="other-tab-list">
```

Comment:
```suggestion
<ul class="other-context-list">
```
IMHO, "tab" is a bit too implementation-specific 😇